tree/tree/inc/TBasket.h

```diff
@@ -123,7 +128,14 @@ class TBasket : public TKey {
            Int_t   ReadBasketBytes(Long64_t pos, TFile *file);
    virtual void    Reset();
 
-           Int_t   LoadBasketBuffers(Long64_t pos, Int_t len, TFile *file, TTree *tree = 0);
+// Time spent reseting basket sizes (typically, at event cluster boundaries), in microseconds
+#ifdef R__TRACK_BASKET_ALLOC_TIME
+   ULong64_t ResetAllocationTime() const { return fResetAllocationTime; }
+#endif
+   // Count of resets performed of basket size.
+   Bool_t ResetAllocationCount() const { return fResetAllocation; }
```

Comment:
The name of the routine would mean 'reset the allocation count', consider prefix this (and above) with Get (as in TTree).